app/model/collection.py

```diff
@@ -14,7 +14,15 @@ class CollectionReadDTO(BaseModel):
 class CollectionWriteDTO(BaseModel):
     """Representation of a Collection for writing."""
 
-    import_id: str
     title: str
     description: str
     organisation: str
+
+
+class CollectionCreateDTO(BaseModel):
+    """Representation of a Collection for creating."""
+
+    title: str
+    description: str
+    # families: list[str] TODO: Ask Patrick if we want this as an option?
```

Comment:
Have we confirmed with Patrick whether or not we want this option?

Reply:
Not yet ... but I tend to litter TODO's around to keep velocity - keeping a task before release that we go through and remove / ticket / fix them :)